src/views/ProfileCreation/Mint.tsx

```diff
@@ -37,6 +40,22 @@ const Mint: React.FC = () => {
   const hasMinimumCakeRequired = fetchStatus === FetchStatus.SUCCESS && cakeBalance.gte(MINT_COST)
   const { callWithGasPrice } = useCallWithGasPrice()
 
+  useEffect(() => {
+    const getStarterNfts = async () => {
+      const { data: allPbTokens } = await getNftsFromCollectionApi(pancakeBunniesAddress)
+      const nfts = STARTER_NFT_BUNNY_IDS.map((bunnyId) => {
+        if (allPbTokens && allPbTokens[bunnyId]) {
+          return { ...allPbTokens[bunnyId], bunnyId }
+        }
+        return undefined
```

Comment:
@0xCorgi wdyt?

I want to prevent an app crash if the the fetch fails - which would happen if you attempted something like 

```
// data = null
data['5']
```

This solution seems a little inelegant though

Reply:
Not sure it's that much better but we could do that:

```suggestion
     STARTER_NFT_BUNNY_IDS
        .filter(bunnyId => !allPbTokens?[bunnyId])
        .map(bunnyId => { ...allPbTokens[bunnyId], bunnyId })
```

Reply:
Akh yea cheers. Let's leave it as is.